kernel/rtlil.cc

```diff
@@ -280,6 +280,32 @@ int RTLIL::Const::as_int(bool is_signed) const
 	return ret;
 }
 
+void RTLIL::Const::compress(bool is_signed)
```

Comment:
What's the motivation for not respecting/setting `flags 
& CONST_FLAG_SIGNED` ?

Reply:
I am not sure we want to defer to that. That flag isn't set to a useful value in all contexts, especially when you convert a `SigSpec` with `as_const()`, which is the use case which prompted writing the helper.